db/seeds.rb

```diff
@@ -5,3 +5,21 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+Call.destroy_all
+Case.destroy_all
+Patient.destroy_all
+
+10.times do |i|
+  Patient.create({name: "Patient #{i}", primary_phone: "123-123-1234"})
```

Comment:
It rules that you started the seed file!

What do you think about mixing up the primary phone number a little bit? Generally when we get a new patient we first check their phone number to make sure they aren't in our system already, so phone is kinda considered a unique key for the users. (Let me know if that doesn't make sense! A look at some sample DCAF data from the excel sheet might be helpful here.)

tl;dr: What do you think about making `primary_phone` be "123-123-123#{i}" so each of the 10 sample patients here has a unique phone number?